influxdb-init/README.md

```diff
@@ -0,0 +1,8 @@
+# Influx Init
+
+# Variables
+INFLUXDB_DEFAULT_RETENTION - determines how long the data should be stored before it is dropped (default for all series created on the database) https://docs.influxdata.com/influxdb/v1.1/query_language/database_management/#retention-policy-management
```

Comment:
Put variables like INFLUXDB_DEFAULT_RETENTION in back ticks `INFLUXDB_DEFAULT_RETENTION`.
Too long lines, split them around 80-100 column, please.
Put long links at the end of file and link to them from here (you could check how to do it in [Kafka-init README](https://raw.githubusercontent.com/monasca/monasca-docker/master/kafka-init/README.md) but you could use aliases instead of numbers).
We are using [InfluxDB 1.3](https://github.com/monasca/monasca-docker/blob/master/.env#L1) so link to the same version, please.